includes/blocks/course-theme/class-module-title.php

```diff
@@ -0,0 +1,87 @@
+<?php
+/**
+ * File containing the Module_Title class.
+ *
+ * @package sensei
+ * @since
+ */
+
+namespace Sensei\Blocks\Course_Theme;
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+
+use \Sensei_Blocks;
+
+/**
+ * Display the title of the current module for the current lesson.
+ */
+class Module_Title {
```

Comment:
Just to avoid confusion in the future (when we have the modules page), maybe we could call it (files, classes, block name, CSS classes, and so on.) as "Lesson Module". WDYT?

Reply:
Sure! Renamed it to `Lesson_Module`.